Caleb b auth context ga (#18214)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/authenticationcontextclassreference-get.md

Namespace: microsoft.graph

Retrieve the properties and relationships of a [authenticationContextClassReference](../resources/authenticationcontextclassreference.md) object.

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type                        | Permissions (from least to most privileged)                                       |
|:--------------------------------------|:----------------------------------------------------------------------------------|
|Delegated (work or school account)     | Policy.Read.ConditionalAccess |
|Delegated (personal Microsoft account) | Not supported. |
|Application                            | Policy.Read.ConditionalAccess |

## HTTP request

<!-- { "blockType": "ignored" } -->

```http
GET /identity/conditionalAccess/authenticationContextClassReferences/{id}
```
## Optional query parameters

This method does not support optional query parameters to customize the response.

## Request headers

| Name      |Description|
|:----------|:----------|
| Authorization | Bearer {token} |

> ## Request body
> 
> This method supports the `$select` OData query parameter to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).

Reviewer: Duplicate section.
Your API also doesn't support $select
```suggestion
```

## Response

> If successful, this method returns a `200 OK` response code and the requested [authenticationContextClassReferences](../resources/\authenticationcontextclassreference.md) object in the response body.

Reviewer: ```suggestion
If successful, this method returns a `200 OK` response code and an [authenticationContextClassReference](../resources/authenticationcontextclassreference.md) object in the response body.
```

## Examples

### Request

The following is an example of the request.

> # [HTTP](#tab/http)

Reviewer: ```suggestion
```

Reviewer: Bump - This line will be added by the snippets generator

<!-- {
  "blockType": "request",
  "name": "get_authenticationcontextclassreference"
}-->

> 

Reviewer: Add the HTTP Request here enclosed in backticks.

### Response

The following is an example of the response.

> > **Note:** The response object shown here might be shortened for readability. All the properties will be returned from an actual call.

Reviewer: ```suggestion
> **Note:** The response object shown here might be shortened for readability.
```
